src/Sylius/Bundle/AddressingBundle/Form/Type/ZoneType.php

```diff
@@ -74,6 +82,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     /**
      * {@inheritdoc}
      */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+
+        $resolver->setDefault('options', array('zone_type' => null));
```

Comment:
I think that should be just ``array('zone_type' => null`)``. Actually, any problem with setting country as default zone type? I'd be fine with that. In future where we may consider decoupling Zones that would be wrong but for now that would simplify some stuff.